zombienet/native.toml

```diff
@@ -21,19 +23,45 @@ chain = "local"
 force_decorator = "generic-evm"
 
   [[parachains.collators]]
-  name = "laos"
+  name = "laos0"
   ws_port = 9999
   command = "{{ZOMBIENET_LAOS_COMMAND}}"
   validator = true
-
+  args = ["--log=xcm=trace"]
+  
   [[parachains.collators]]
-  name = "laos-1"
-  ws_port = 10000
+  name = "laos1"
   command = "{{ZOMBIENET_LAOS_COMMAND}}"
   validator = true
+  args = ["--log=xcm=trace"]
 
-  [[parachains.collators]]
-  name = "laos-2"
-  ws_port = 10001
+    [[parachains.collators]]
+  name = "laos2"
   command = "{{ZOMBIENET_LAOS_COMMAND}}"
   validator = true
+  args = ["--log=xcm=trace"]
+
+[[parachains]]
+id = 1000
+cumulus_based = true
+chain = "asset-hub-rococo-local"
+
+  [[parachains.collators]]
+  name = "asset-hub"
+  validator = true
+  ws_port = 9950
+  command = "{{ZOMBIENET_ASSETHUB_COMMAND}}"
+  args = ["--log=xcm=trace,pallet-assets=trace"]
+
+# See: https://github.com/paritytech/polkadot-sdk/pull/1616
+# [[hrmp_channels]]
+# sender = 2900
+# recipient = 1000
+# max_capacity = 8
+# max_message_size = 1048576 # 512
+
+# [[hrmp_channels]]
+# sender = 1000
+# recipient = 2900
+# max_capacity = 8
+# max_message_size = 1048576
```

Comment:
Maybe we can remove these lines as we saw there's this [issue](https://github.com/paritytech/polkadot-sdk/pull/1616) with the zombienet and we're manually open the channels with our test.
```suggestion
```

Reply:
I agree with you. I just suggested to delete it as the issue has been open for more than a year. Anyway as you said it's not a big problem